starter/.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
this shouldn't be necessary since you have this file at the root.

Reply:
I added it on the root, after I added it here, realizing it was needed in other folders. Will Nix.

Reply:
Removed